Src/AutoFixtureUnitTest/Kernel/TerminatingWithPathSpecimenBuilderTest.cs

```diff
@@ -200,18 +224,18 @@ public void SpecimenRequestsAreEmptyAfterThrowing()
             var requests = new[] {new object(), new object(), new object() };
             var requestQueue = new Queue<object>(requests);
             var firstRequest = requestQueue.Dequeue();
-            var tracer = new DelegatingTracingBuilder(new DelegatingSpecimenBuilder
+            var builder = new DelegatingSpecimenBuilder
             {
                 OnCreate = (r, c) => requestQueue.Count > 0
                     ? c.Resolve(requestQueue.Dequeue())
                     : new NoSpecimen()
-            });
-            var sut = new TerminatingWithPathSpecimenBuilder(tracer);
+            };
+            var sut = new TerminatingWithPathSpecimenBuilder(builder);
+            // Cause sut to be executed recursively for multiple times.
```

Comment:
Replied above. Let's discuss there. If action is required - I'll fix both places.